spec/rubocop/cop/style/trailing_underscore_variable_spec.rb

```diff
@@ -224,11 +224,6 @@
 
     include_examples 'common functionality'
 
-    it 'does not register an offense for an underscore variable preceded ' \
-       'by a named splat underscore variable' do
-      expect_no_offenses('a, *_b, _ = foo()')
```

Comment:
This spec duplicated of https://github.com/bbatsov/rubocop/pull/4989/files#diff-7fe7950ba35a008e696293c0f39bf18bL244